client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
```

Comment:
My only problem is that the `getSite` selector is doing a lot and we could easily avoid it here. But that’s not a strong opinion and it feel free to ignore it – it’s not a merge blocker.